### ISSUE TITLE
filterid support removed

### ISSUE THREAD

**Issue by apaperclip** [seanauff/metermon#13]
@jjlawren 

I think pr https://github.com/seanauff/metermon/pull/7 may have wiped out the support for filterid.  

commits:

https://github.com/seanauff/metermon/commit/098d08b7a2e952424b65145dc4f761913643f206#diff-0e6e87e8875f3078961c72e4892d11e8875a595652782d6d711fdbc751e16171
and
https://github.com/seanauff/metermon/commit/e443ea9bb978cf353a3fe2d836e081724073bf77#diff-0e6e87e8875f3078961c72e4892d11e8875a595652782d6d711fdbc751e16171

I'm still looking and testing.

**Comment by jjlawren** [comment]
It's not needed as `rtlamr` reads the environment variables directly. Check your logs for confirmation.

**Comment by apaperclip** [comment]
Thanks for the quick reply, I'll check that out.

**Comment by apaperclip** [comment]
Ok I think I found the issue.

The tuple for SCM+ water meters needs the ID for my itron 100w-r.  0xAB/171.  I added this to my local metermon.py and I started getting the data in stdout and mqtt.

I'll do a PR tonight

**Comment by seanauff** [comment]
Merged the PR and pushed new images to DockerHub. Thanks for figuring this out!

**Comment by apaperclip** [comment]
you are welcome!
If you start getting more of these requests for higher values, there may be a more sustainable option.

I learned when making the related PR to rtlamr , the SCM+ endpoint type value has more 'stuff' in it.  In my case  0xAB / 171 is actually type 11, not type AB. This fits the existing defined water endpoint type.    See rtlamr pr thread [here](https://github.com/bemasher/rtlamr/pull/192).  I noticed 181 was previously added as gas id,  it's probably more likely  other bits and then 5.
